ingestion/src/metadata/ingestion/source/pipeline/airflow/metadata.py

```diff
@@ -278,10 +300,14 @@ def get_pipelines_list(self) -> Iterable[OMSerializedDagDetails]:
                     description=data.get("_description", None),
                     start_date=data.get("start_date", None),
                     tasks=data.get("tasks", []),
-                    owners=data.get("default_args", [])["__var"].get("email", [])
-                    if data.get("default_args")
-                    else None,
+                    schedule_interval=self.get_schedule_interval(data),
                 )
+                if self.source_config.includeOwners:
+                    dag.owners = (
+                        data.get("default_args", [])["__var"].get("email", [])
```

Comment:
instead of having this `dag.owners` set here, we could have it in a function and add it where we are directly initializing the `AirflowDagDetails`